app/javascript/components/track/approaches-elements/ArticleSnippet.tsx

```diff
@@ -16,8 +16,12 @@ export function ArticleSnippet({ article }: { article: Article }): JSX.Element {
         ref={codeBlockRef}
       >
         <div
-          className="c-cli-walkthrough"
+          className="c-cli-walkthrough overflow-hidden block"
```

Comment:
Why is this using cli-walkthrough as a class?

Reply:
I don't know it doesn't really have any effect here. @ErikSchierboom?

Reply:
I don't think I created that file, so 🤷